src/games/_gameswidget.py

```diff
@@ -28,20 +28,18 @@ class GamesWidget(FormClass, BaseClass):
     sub_factions = Settings.persisted_property(
         "play/subFactions", default_value=[False, False, False, False])
 
-    def __init__(self, client, game_model, me,
-                 gameview_builder, game_launcher):
+    def __init__(self, client, game_model, me, gameview_builder, game_launcher):
         BaseClass.__init__(self)
-
         self.setupUi(self)
 
         self._me = me
         self.client = client
         self.mods = {}
-        self._gameview_builder = gameview_builder
+        #self._gameview_builder = gameview_builder
```

Comment:
Remove code instead of commenting it out.

Reply:
ok. (I left it for a reason, but with this view .. not needed)